Examples/Python/tests/root_file_hashes.txt

```diff
@@ -42,11 +42,11 @@ test_propagation__propagation_steps.root: b5da12754f97b67dce6bf85a7345481ec5bebc
 test_particle_gun__particles.root: 78a89f365177423d0834ea6f1bd8afe1488e72b12a25066a20bd9050f5407860
 test_digitization_example__measurements.root: f2dd54cd8315e4656136571d4802a69293d7feb71f427706410b8f0d2ad76265
 
-test_material_recording__geant4_material_tracks.root: 166dfb9f7a0a19d2206cf7c5ada02189e0b04a49ec7c25a59007e3ee23a62035
-test_material_mapping__material-map_tracks.root: 80beaaf4b9acee9eef6f2bb2094d1c819edb07f22e949833a0dc4143a697ad32
-test_material_mapping__propagation-material.root: 15d5dfd89583b965ec5a2625126a190b4af92f21b4f2f9ce2946e98bf8dff34a
-test_volume_material_mapping__material-map-volume_tracks.root: 14815e3f42a64c140450302f06a4381ede147ee4edbb8a88f3dfac7e7ab53fa7
-test_volume_material_mapping__propagation-volume-material.root: 47b488e258ca4c964ba9f68a65e4ea576bc096f2c8476b1363b9f98b5cc96820
+test_material_recording__geant4_material_tracks.root: 69d3b13e800615f38fcfef9bf2c6ac1ec799f2d3edba62954be668fa44282cf1
+test_material_mapping__material-map_tracks.root: efe7ce049e1b5257f37485f6788c3e67e83bb3ca987dd0e0bc34edb9cc2a6c0d
+test_material_mapping__propagation-material.root: e3677926877a57a87cb6db59cf6e3016146e5c7baf4b67fd65424a070502bc65
+test_volume_material_mapping__material-map-volume_tracks.root: ecd77adfe5421218df55d07842591f2721e9e130ad55e71bb0da94977a74dfe7
+test_volume_material_mapping__propagation-volume-material.root: e3a65793321cc2ec901bce83df068ef5e364bf5091beec9a3f80b5ab6f8c5d87
```

Comment:
I assume you checked that the changed outputs look reasonable?

Reply:
Yes and I have been running the material mapping with the new change for a while. They give the same result as before. The difference here come from the extra information I added to the root file.